sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 1.0.0-beta.11 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.0-beta.11 (2024-09-12)
 
 ### Other Changes
 
+- Updated the internal AMQP implementation with the latest fixes and improvements.
```

Comment:
@LarryOsterman, you probably have better wording for this, if so - please share! Maybe it will be better to have something for the "Bugs Fixed" section instead of "Other Changes" which gets removed from the release highlights (for example, https://azure.github.io/azure-sdk/releases/2024-08/cpp.html does not have any of the "Other Changes" sections from Core).

Reply:
I just submitted a PR for this.

Reply:
Thank you, let's get it merged, and I'll update this PR.